### ISSUE TITLE
Test disabling editor commenting

### ISSUE THREAD

**Issue by alexr00** [microsoft/vscode#153314]
Refs: https://github.com/microsoft/vscode/issues/149176

- [x] anyOS @TylerLeonhardt
- [x] anyOS @brettcannon

Complexity: 3

[Create Issue](https://github.com/microsoft/vscode/issues/new?body=Testing+%23153314%0A%0A&assignees=alexr00)

---

1. Install an extension that provides comments, such as GitHub Pull Requests and Issues and get into a state where there are some comments and commenting ranges.
  a. If you're using GitHub Pull Requests and Issues, then you can checkout a pull request and leave some comments to get into a good state for testing
2. Run the "Comments: Toggle Editor Commenting Command" and verify:
  a. that all commenting UX (commenting ranges and comment widgets) is removed from all editors
  b. that you can toggle commenting back on
  c. that if you open the comments view and click on a comment that the comment will show in the editor
3. Toggle commenting back on, toggle zen mode on and verify:
  a. that editor commenting is off by default in zen mode
  b. that you can toggle editor commenting on in zen mode

**Comment by TylerLeonhardt** [comment]
Works exactly as expected :)

**Comment by alexr00** [comment]
@brettcannon that should be fixed in the latest insiders (clicking on a comment in the view toggles commenting back on).

**Comment by brettcannon** [comment]
I've verified I'm on v0.45.2022062909 today. I thought I checked there wasn't an update, but I was also admittedly exhausted yesterday, very possibly PEBKAC. 😅

Only thing I'm now seeing is two clicks are required to bring a line into view and then to show the comment.
![screen-capture](https://user-images.githubusercontent.com/54418/176502753-e65e4636-0a39-4ef6-ac8e-53e750392fcb.gif)

I'm assuming that's on purpose or some design limitation, so I'm closing as LGTM!
